src/data-table/index.js

```diff
@@ -43,7 +46,7 @@ class DataTable extends Component {
     this.state = {
       sortColumn,
       activePage: 1,
-      sortDirection: 'descending',
+      sortDirection: props.defaultSortDirection,
```

Comment:
can we do this like in line 44 to use destructured (but renamed into `sortDirection` like `sortColumn`), so similar steps are performed in one place?